terraform/deployments/cluster-services/argo.tf

```
@@ -85,6 +78,18 @@ resource "helm_release" "argo_cd" {
     dex = {
       enabled = false
     }
+
+    notifications = {
+      cm = {
+        create = false
+      }
+
+      "argocdUrl" = "https://${local.argo_host}"
+
+      secret = {
+        create = false
+      }
```

nit: no need for the quotes around the key there. Also can make this a bit more concise like this: (also in alphabetical order cos why not) 
```suggestion
    argocdUrl = "https://${local.argo_host}"
    cm        = { create = false }
    secret    = { create = false }
```

---

cool thanks, I thought about the first one and then I forgot :-(